src/hooks/useCart/useCart.test.tsx

```diff
@@ -90,7 +90,12 @@ const defaultProducts: EnvVersion = {
   features: {
     REQUIRE_OTP: true,
     TRANSACTION_GROUPING: true,
-    FLOW_TYPE: "DEFAULT"
+    FLOW_TYPE: "DEFAULT",
+    id: {
```

Comment:
Lol super small nit, but the caps is different. HAHAHA. It's ok we can change later, not a must this PR 😂

Reply:
😂 yes it's quite a sore eye! i'll just change it in this PR bah? then remove the redundant `REQUIRE_OTP` flag as well!

Reply:
HAHAA agree, super eye sore. I'll create a card for this changes + removing `REQUIRE_OTP` since its unused!

Reply:
I think can leave it first? Once the campaign config parts are up, we'll use the normal camelCase convention.

Reply:
haha okk~~ sounds good!

Reply:
alright, shall leave it for now!